src/pages/Login/LoginPage.tsx

```diff
@@ -0,0 +1,111 @@
+import { IonButton, IonContent, IonInput, IonItem, IonLabel, IonPage, IonText, IonToast } from '@ionic/react';
+import { useHistory } from 'react-router';
+import { useCallback, useState } from 'react';
+
+import LoginPageStyles from './LoginPage.module.scss';
+
+const LoginPage: React.FC = () => {
+  const history = useHistory();
+  const initializationError = {
+    // load this information from store in case the login is not working due to firebase(config) issues
+    message: 'Fehler beim Verbindungsaufbau zur Datenbank',
+  };
+  const [tenant, setTenant] = useState(''); // organization, account, subaccount, tenant
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorInfo, setErrorInfo] = useState({ showErrorToast: false, errMsg: '' });
+
+  const login = useCallback(() => {
+    // todo: remove when connecting to authentication service
+    console.log(`Tenant: ${tenant}`);
+    console.log(`E-Mail: ${email}`);
+    console.log(`Passwort: ${password}`);
+    // todo: validate email for correct email format
+    setErrorInfo({ showErrorToast: true, errMsg: 'Benutzername und Passwort stimmen nicht überein' });
+  }, [tenant, email, password]);
+
+  const handleLogin = useCallback(
+    (e: React.MouseEvent<HTMLIonButtonElement, MouseEvent>) => {
+      if (!e.currentTarget) {
+        return;
+      }
+      e.preventDefault();
+      login();
+    },
+    [login],
+  );
+
+  return (
+    <IonPage className={LoginPageStyles.page}>
+      <IonContent className={LoginPageStyles.content}>
+        <IonText color="danger" style={{ fontWeight: '500' }}>
+          {initializationError && initializationError.message}
+        </IonText>
+
+        <img src="/assets/icon/icon.png" alt="Team-Manager Logo" className={LoginPageStyles.logo} />
+
+        <p className={LoginPageStyles.loginTitle}>Anmeldung</p>
+
+        <IonItem className={LoginPageStyles.input}>
+          <IonLabel position="stacked">Organisations-Id deines Teams</IonLabel>
+          <IonInput
+            type="email"
+            placeholder="Organisation"
+            onIonChange={(e) => {
+              if (e && e.detail && e.detail.value) {
+                setTenant(e.detail.value);
+              }
+            }}
```

Comment:
Define callback like you did for handle Login.

```suggestion
            onIonChange={handleIonChange}
```

Reply:
handleIonChange as a generic handler function seems not to be applicable, since we're not able to pass any information about the changed property. The event only contains detail and value, no further information about the input field surround it (despite we'd check the DOM element).

Another problem that occurs on my end is the CustomEvent type that can not be resolved and leads to an error. See attached.
![image](https://user-images.githubusercontent.com/22232458/108588697-d4d65800-735a-11eb-9c77-7d5575b47bee.png)

![image](https://user-images.githubusercontent.com/22232458/108588703-dacc3900-735a-11eb-8b39-57f025445eb1.png)